### DISCUSSION TITLE
Tutorial step 14 config file error?

### DISCUSSION THREAD

**Discussion by ttrickel** [missionpinball/missionpinball#70] [Q&A] [answered]
I'm running MPF v0.55.0 and when I run MPF using the base mode config file:

##! mode: my_mode
#config_version=5
mode:
  start_events: ball_started
  priority: 100

I get an error that ends with:

ValueError: Error found in file C:\Users\student\pinball\cobraPinTest1\modes\base\config\base.yaml: Version mismatch. Expected: #config_version=5 Actual: ##! mode: base Files: C:\Users\student\pinball\cobraPinTest1\modes\base\config\base.yaml

Further I find that mpf -version does not work on MPF v0.55.0 as stated on the [MPF Versions](https://docs.missionpinball.org/en/latest/versions/index.html?highlight=update%20mpf#mpf-versions) page.

I should probably update MPF but I'm worried about the Windows sound issue.

**Comment by Pucksnok** [comment, marked as answer]
Always use the #config_version=5 at the top of your yaml file.

**Reply by ttrickel** [reply]
Thanks Pucksnok,

I edited the doc page to reflect that and submitted a pull request.

**Comment by ttrickel** [comment]
Thanks Gill,

I edited the dic page to reflect that.

**Reply by Stefku** [reply]
`##! mode: my_mode` is an indication to which file the example belongs. And you're right, problem is it's not conform to the syntax...
